server/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -1159,9 +1158,6 @@ private Index(final StreamInput in) throws IOException {
             }
             source = in.readBytesReference();
             routing = in.readOptionalString();
-            if (format < FORMAT_NO_PARENT) {
```

Comment:
Removing this old dead code here since it's not worth a separate PR, we never set the format variable to something before the no parent format these days.